src/requirements/data/majors/info.ts

```diff
@@ -58,8 +58,208 @@ const infoRequirements: readonly CollegeOrMajorRequirement[] = [
       'Students are required to complete AT LEAST one concentration from the seven concentrations available',
     source:
       'https://infosci.cornell.edu/undergraduate/info-sci-majors/ba-information-science-college-arts-sciences/degree-requirements-0',
-    fulfilledBy: 'self-check',
-    minCount: 1,
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      'Behavioral Science': {
+        description:
+          'A. Understanding Social Behavior (choose two), ' +
+          'B. Social Data Analytics (choose one), ' +
+          'C. Behavior in Sociological, Network, and Design Contexts (choose one)',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          [
+            'INFO 3400',
+            'INFO 3460',
+            'INFO 4430',
+            'INFO 4450',
+            'INFO 4500',
+            'INFO 4800',
+            'INFO 4940',
+            'COMM 4380',
+            'PSYCH 3800',
+          ],
+          ['INFO 3300', 'INFO 4100', 'INFO 4300', 'COMM 4242', 'CS 4740', 'CS 4780'],
+          [
+            'INFO 3200',
+            'INFO 3561',
+            'INFO 4650',
+            'COMM 4410',
+            'STS 3440',
+            'INFO 4360',
+            'SOC 3350',
+            'INFO 3450',
+            'INFO 4240',
+            '​INFO 4400',
+          ]
+        ),
+        perSlotMinCount: [2, 1, 1],
+      },
+      'Data Science': {
+        description:
+          'A. Data Analysis (choose one), ' +
+          'B. Domain Expertise (choose one), ' +
+          'C. Big Data Ethics, Policy and Society (choose one), ' +
+          'D. Data Communication (choose one)',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['INFO 3300', 'INFO 3950', 'CS 4780', 'CS 4786', 'ORIE 3120', 'ORIE 4740', 'STSCI 4740'],
+          ['INFO 2770', 'INFO 3350', 'INFO 4100', 'INFO 4120', 'INFO 4130', 'INFO 4300', 'CS 4740'],
+          [
+            'INFO 3200',
+            'INFO 3561',
+            'INFO 4200',
+            'INFO 4240',
+            'INFO 4250',
+            'INFO 4270',
+            'INFO 4561',
+            '​INFO 4940',
```

Comment:
INFO 4940 doesn't appear in the See All for this slot (third slot or C. Big Data Ethics) of the Data Science concentration. Lol I noticed this bc I was trying to track my Data Science concentration fulfillment.

Reply:
Interesting I get this when I try to add INFO 4940 from the add modal
![Screen Shot 2021-03-15 at 10 08 19 AM](https://user-images.githubusercontent.com/54298311/111166373-6b6ef100-8576-11eb-9cec-6b30cc60f569.png)

Reply:
This also happened to me ^ but I see it in the Add Modal/See All for the Behavioral Sciences Concentration

Reply:
I still can't seem to find INFO 4940 for the Data Science concentration. I don't think the zero-width space for this was removed in your commit.

Also, I wonder why CALS Credits no longer shows up as a possible option for INFO 4940 in the add modal. (This is the add modal I get when I'm an A&S INFO major Data Science concentration).
![Screen Shot 2021-03-15 at 7 37 45 PM](https://user-images.githubusercontent.com/54298311/111234895-eca1a480-85c5-11eb-8570-4f9b06b81f0a.png)